src/ChannelAudience/RoleAudience.js

```diff
@@ -0,0 +1,17 @@
+const ChannelAudience = require('../ChannelAudience');
+
+class RoleAudience extends ChannelAudience {
+  constructor(options) {
+    super(options);
+    if (!options.hasOwnProperty('minRole')) {
+      throw new Error('No role given for role audience');
+    }
+    this.minRole = options.minRole;
+  }
+
+  getBroadcastTargets() {
+    return [...this.state.PlayerManager.players].filter(player => player.role >= this.minRole);
```

Comment:
looking at the other channel audience this line can just be `this.state.PlayerManager.filter`  instead of directly accessing `PlayerManager.player`